lib/modules/manager/pip_requirements/common.ts

```diff
@@ -33,13 +33,23 @@ export function extractPackageFileFlags(
   const additionalRequirementsFiles: string[] = [];
   const additionalConstraintsFiles: string[] = [];
   content.split(newlineRegex).forEach((line) => {
-    if (line.startsWith('-i ') || line.startsWith('--index-url ')) {
-      registryUrls = [line.split(' ')[1]];
-    } else if (line.startsWith('--extra-index-url ')) {
-      const extraUrl = line
-        .substring('--extra-index-url '.length)
-        .split(' ')[0];
-      additionalRegistryUrls.push(extraUrl);
+    if (line.startsWith('-i ') || line.startsWith('--index-url')) {
+      // parameters can be separated by space or =, same below for --extra-index-url
+      if (line.includes('=')) {
+        registryUrls = [line.split('=')[1]];
```

Comment:
Due to split on `=` the flags will be included in the registryUrl too 
ie. for
```
--index-url=https://artifactory.company.com/artifactory/api/pypi/python/simple --trusted-host artifactory.company.com --default-timeout 600
```

registryUrl will be:
`
https://artifactory.company.com/artifactory/api/pypi/python/simple --trusted-host artifactory.company.com --default-timeout 600
`

whereas we want:
`
https://artifactory.company.com/artifactory/api/pypi/python/simple
`

This is also the cause of the failing tests in CI.

Reply:
Ah, good point. I'll look into it as as find time.